internal/checks/source-code/repository-management/repository_management_test.go

```diff
@@ -22,130 +22,52 @@ func TestRepositoryChecker(t *testing.T) {
-				checkmodels.ToCheckRunResult("1.2.4", checksMetadata.Checks["1.2.4"], checksMetadata.Url, &checkmodels.CheckResult{Status: checkmodels.Passed}),
-			},
-		},
-		{
-			Name: "Public repository with security.md file",
-			Data: &checkmodels.CheckData{
-				AssetsMetadata: builders.NewAssetsDataBuilder().WithOrganization(builders.NewOrganizationBuilder().WithReposDeletionLimitation(true).WithMembersCanCreateRepos(true).WithIssuesDeletionLimitation(true).Build()).
-					WithRepository(builders.NewRepositoryBuilder().WithPrivate(false).WithSecurityMdFile(true).Build()).Build(),
-			},
-			Expected: []*checkmodels.CheckRunResult{
-				checkmodels.ToCheckRunResult("1.2.1", checksMetadata.Checks["1.2.1"], checksMetadata.Url, &checkmodels.CheckResult{Status: checkmodels.Passed}),
-				checkmodels.ToCheckRunResult("1.2.2", checksMetadata.Checks["1.2.2"], checksMetadata.Url, &checkmodels.CheckResult{Status: checkmodels.Failed}),
-				checkmodels.ToCheckRunResult("1.2.3", checksMetadata.Checks["1.2.3"], checksMetadata.Url, &checkmodels.CheckResult{Status: checkmodels.Passed}),
-				checkmodels.ToCheckRunResult("1.2.4", checksMetadata.Checks["1.2.4"], checksMetadata.Url, &checkmodels.CheckResult{Status: checkmodels.Passed}),
-			},
-		},
-		{
-			Name: "Private repository with security.md file",
-			Data: &checkmodels.CheckData{
-				AssetsMetadata: builders.NewAssetsDataBuilder().WithOrganization(builders.NewOrganizationBuilder().WithReposDeletionLimitation(true).WithMembersCanCreateRepos(true).WithIssuesDeletionLimitation(true).Build()).
-					WithRepository(builders.NewRepositoryBuilder().WithPrivate(true).WithSecurityMdFile(true).Build()).Build(),
-			},
-			Expected: []*checkmodels.CheckRunResult{
-				checkmodels.ToCheckRunResult("1.2.1", checksMetadata.Checks["1.2.1"], checksMetadata.Url, &checkmodels.CheckResult{Status: checkmodels.Passed}),
-				checkmodels.ToCheckRunResult("1.2.2", checksMetadata.Checks["1.2.2"], checksMetadata.Url, &checkmodels.CheckResult{Status: checkmodels.Failed}),
-				checkmodels.ToCheckRunResult("1.2.3", checksMetadata.Checks["1.2.3"], checksMetadata.Url, &checkmodels.CheckResult{Status: checkmodels.Passed}),
-				checkmodels.ToCheckRunResult("1.2.4", checksMetadata.Checks["1.2.4"], checksMetadata.Url, &checkmodels.CheckResult{Status: checkmodels.Passed}),
 			},
 		},
 		{
-			Name: "Repository without stale branches",
+			Name: "Repository without creation limited to trusted users",
 			Data: &checkmodels.CheckData{
 
 				AssetsMetadata: builders.NewAssetsDataBuilder().
-					WithOrganization(builders.NewOrganizationBuilder().WithReposDeletionLimitation(true).WithIssuesDeletionLimitation(true).WithMembersCanCreateRepos(true).Build()).
-					WithRepository(builders.NewRepositoryBuilder().WithPrivate(true).WithSecurityMdFile(false).Build()).Build(),
+					WithOrganization(builders.NewOrganizationBuilder().WithMembersCanCreateRepos(true).Build()).Build(),
```

Comment:
nice! thats how it should be - sending the builder only what you need for the test case, the rest - doesn't matter and live inside of the initial object.